BackendApartmentReservation/BackendApartmentReservation/Repositories/Checklist/HotelRepository.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BackendApartmentReservation.Database;
+using BackendApartmentReservation.Database.Entities.Amenities;
+using BackendApartmentReservation.Database.Entities.Reservations;
+using BackendApartmentReservation.Repositories.Checklist;
+using Microsoft.EntityFrameworkCore;
+
+namespace BackendHotelReservation.Repositories.Checklist
+{
+    public class HotelRepository : IHotelRepository
+    {
+
+        private readonly DatabaseContext _db;
+
+        public HotelRepository(DatabaseContext db)
+        {
+            _db = db;
+        }
+
+        public async Task<DbHotelAmenity> CreateHotelAmenity(string address)
```

Comment:
Not sure if thats a mock, but we should definitely use ID as lookup key

Reply:
As discussed in Slack, disregard this comment